jsapp/js/components/account/accountSidebar.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import {RouteComponentProps} from 'react-router'
+import {ROUTES} from 'js/router/routerConstants'
+import bem from 'js/bem'
+import LoadingSpinner from 'js/components/common/loadingSpinner'
+import Icon from 'js/components/common/icon'
+import './accountSidebar.scss'
+
+const ACCOUNT_SETTINGS_HREF = '#' + ROUTES.ACCOUNT_SETTINGS
+const DATA_STORAGE_HREF = '#' + ROUTES.DATA_STORAGE
+const SECURITY_HREF = '#' + ROUTES.SECURITY
```

Comment:
I think you can just use `'#' + ROUTES.SECURITY` directly as value, no need to create const